in_toto/in_toto_keygen.py

```diff
@@ -30,6 +30,12 @@
 
   python in_toto_keygen.py -p bob_keys 2048
 
+
+<Return Codes>
+  2 if an exception occurred during argument parsing
+  1 if an exception occurred
+  0 if no exception occurred
+
```

Comment:
I don't think it's necessary, since this follows convention.

Reply:
Which convention?

I think documenting return/status codes is standard practice.

Examples:
[HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)
[FTP return codes](https://en.wikipedia.org/wiki/List_of_FTP_server_return_codes)
[Pylint exit codes](https://docs.pylint.org/en/1.6.0/run.html#exit-codes)

Since you only really return 1-2 for exceptions, perhaps you can say that the script returns a non-zero value for any exception.

Reply:
Shell command exit codes follow the convention of returning non-zero in case of failure and zero in case of success.
However, documentation is provided in the form of above docstring.